### ISSUE TITLE
Clicking 'Not Now' in the popped dialog displays '…connected to log-streaming service'after starting streaming logs for one deployment slot

### ISSUE THREAD

**Issue by v-haiz** [Microsoft/vscode-azureappservice#391]
Repro steps:
 1. Create a App and choose Deployment Slots node.
 2. Right click Deployment Slots and create a deployment slot.
 3. Right click the created deployment slot and choose 'Start Streaming Logs'.
 4. Select 'Not Now' in the popped dialog.
 5. Check the log message in output window.

Expect:
The deployment slot isn't connected to log-streaming service.

Actual:
The deployment slot displays '…connected to log-streaming service' in output window.
![actual](https://user-images.githubusercontent.com/37653604/38546201-8bb70eba-3c9b-11e8-8815-ee866fc19e74.png)

**Comment by nturinski** [comment]
@v-haiz Thank you for reporting the issue.  This is actually by design though I think that this is confusing.  Enabling application logging and log streaming are two different actions.  Enabling application logging requires restarting the container which is why we prompt the user if they want to enable it, but it is not required to stream the logs.

@fiveisprime Do you think we should change the user experience?

**Comment by v-haiz** [comment]
Thank you to let me know the reason of this issue more clearly. @nturinski @fiveisprime

**Comment by EricJizbaMSFT** [comment]
Assigning to @fiveisprime. Please draft the message and assign back to @nturinski

**Comment by fiveisprime** [comment]
Can we set the app setting when the app is created and get rid of the message entirely? Are there any downsides to this?

**Comment by nturinski** [comment]
As far as I can tell, this isn't an app setting that we can just edit.  We could do the API call to Azure on creation, but I worry that it wouldn't work immediately after creation, similar to deployment.

Also when we set the app for application logging, we enable File System Logging.  According to the docs:

> For Application logging, you can turn on the file system option temporarily for debugging purposes. This option turns off automatically in 12 hours. You can also turn on the blob storage option to select a blob container to write logs to.

So I don't think there would be a way to ensure that the user never saw this prompt even if we did enable it by default.
